main/modes/flight/flight.c

```diff
@@ -666,6 +666,7 @@ static void flightStartGame(flightModeScreen mode)
     flight->wintime           = 0;
     flight->speed             = 0;
 
+    accelSetRegistersAndReset();
```

Comment:
In hindsight, this should probably be called by the system, not individual modes.